packages/compass-components/src/components/document-list/element-editors.tsx

```diff
@@ -323,6 +323,20 @@ const typeEditorActive = css({
   paddingLeft: 0,
 });
 
+const typeEditorChevron = css({
```

Comment:
fix the type dropdown to a specific height, preventing it from jumping around on hover